Sources/SwiftSyntaxBuilderGeneration/Templates/BuildableCollectionNodesFile.swift

```diff
@@ -0,0 +1,257 @@
+            )
+          ]
+        )
+      ) {
+        SequenceExpr(elements: ExprList([
+          "elements",
+          AssignmentExpr(),
+          FunctionCallExpr(MemberAccessExpr(base: "lists", name: "flatMap"), trailingClosure: ClosureExpr {
+            CodeBlockItem(item: MemberAccessExpr(
+              base: FunctionCallExpr(MemberAccessExpr(base: "$0", name: "create\(type.buildable)")),
+              name: "elements"
+            ))
+          }),
+        ]))
+      }
+
+      InitializerDecl(
+        modifiers: [TokenSyntax.public],
+        parameters: ParameterClause(
+          parameterList: [
+            FunctionParameter(
+              firstName: .identifier("arrayLiteral").withTrailingTrivia(.space),
+              secondName: .identifier("elements"),
+              colon: .colon,
+              type: elementType.expressibleAs,
+              ellipsis: .ellipsis
+            )
+          ]
+        )
+      ) {
+        FunctionCallExpr(MemberAccessExpr(base: "self", name: "init")) {
+          TupleExprElement(expression: "elements")
+        }
+      }
+
+      FunctionDecl(
+        modifiers: [TokenSyntax.public],
+        identifier: .identifier("build\(type.baseName)"),
+        signature: FunctionSignature(
+          input: createFormatLeadingTriviaParameters(withDefaultTrivia: true),
+          output: type.syntax
+        )
+      ) {
+        VariableDecl(letOrVarKeyword: .let) {
+          PatternBinding(
+            pattern: "result",
+            initializer: InitializerClause(
+              value: FunctionCallExpr(MemberAccessExpr(base: "SyntaxFactory", name: "make\(type.baseName)")) {
+                if elementType.isToken {
+                  TupleExprElement(expression: "elements")
+                } else {
+                  TupleExprElement(
+                    expression: FunctionCallExpr(MemberAccessExpr(base: "elements", name: "map"),
+                    trailingClosure: ClosureExpr {
+                      FunctionCallExpr(MemberAccessExpr(base: "$0", name: "build\(elementType.baseName)")) {
+                        TupleExprElement(label: "format", expression: "format")
+                        TupleExprElement(
+                          label: "leadingTrivia",
+                          expression: node.elementsSeparatedByNewline
+                            ? SequenceExpr(elements: ExprList([
+                                MemberAccessExpr(base: "Trivia", name: "newline"),
+                                BinaryOperatorExpr("+"),
+                                FunctionCallExpr(MemberAccessExpr(base: "format", name: "_makeIndent")),
+                              ]))
+                            : "nil"
+                        )
+                      }
+                    })
+                  )
+                }
+              }
+            )
+          )
+        }
+        IfStmt(
+          conditions: ConditionElementList([
```

Comment:
Would it be worth making adding a `Array: ConditionElementList` conformance? 

Also I think we should make `OptionalBindingCondition: ExpressibleAsConditionElement`. Combined, that should allow us to write `conditions: OptionalBindingCondition`.

Reply:
-> #525

This should get us the `OptionalBindingCondition` -> `ConditionElement` -> `ConditionElementList` conformance path. The `Array` conformance already exists AFAICT.